app/views/about-your-organisation/index.html

```diff
@@ -9,6 +10,30 @@
   }) }}
 {% endblock %}
 
+{%- set contactAddressHtml -%}
+  {%- if data["building-and-street"] or data["building-and-street-2"] or data["town-or-city"] or data["county"] or data["postcode"] -%}
+    {{- data["building-and-street"] + "\n" if data["building-and-street"] -}}
+    {{- data["building-and-street-2"] + "\n" if data["building-and-street-2"] -}}
+    {{- data["town-or-city"] + "\n" if data["town-or-city"] -}}
+    {{- data["county"] + "\n" if data["county"] -}}
+    {{- data["postcode"] if data["postcode"] -}}
+  {%- else -%}
+    Empty
+  {%- endif -%}
+{%- endset -%}
+
+{% set placementPolicyHtml %}
+  {% if data["placement-policy"] == "hosted" %}
+    Candidates choose a lead or host school
+  {% elif data["placement-policy"] == "placed" %}
+    Placements are arranged with candidates before their training starts
+  {% elif data["placement-policy"] == "other" %}
+    Other
```

Comment:
I wonder if these options don't seem mutually exclusive enough - eg because placements might be arranged with candidates post-application, but they might still get _some_ choice in the school (which seems common)? Similarly, even if they choose a lead school (when they apply), the placement is still 'arranged' later?

Hard to reword though! Vague suggestion below:

```suggestion
  {% if data["placement-policy"] == "hosted" %}
    Candidates choose a lead school as part of their application
  {% elif data["placement-policy"] == "placed" %}
    School placements are arranged with candidates later
  {% elif data["placement-policy"] == "other" %}
    Other
```